src/builder.rs

```diff
@@ -60,6 +61,26 @@ enum PositionState {
     Set,
 }
 
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum BuilderError {
+    #[error("Builder position is not set")]
+    PositonError,
```

Comment:
typo: Position

Reply:
Maybe `UnsetPosition` would be a better name? It's sorta redundant to have Error in the variant name

Reply:
True. Changing that now - and I will also run `typos` over Inkwell. Perhaps adding `typos` can be added to CI?

Reply:
I didn't know about `typos` before - could you open a ticket please?

Reply:
Yes, will open it now.